pkg/cluster/cluster.go

```diff
@@ -410,11 +414,36 @@ func (c *Cluster) delete() {
 	if c.spec.Backup != nil {
 		k8sutil.DeleteBackupReplicaSetAndService(c.kclient, c.name, c.namespace, c.spec.Backup.CleanupOnClusterDelete)
 	}
+
+	err = c.deleteClientServiceLB()
+	if err != nil {
+		// todo: do not panic!
+		panic(err)
+	}
+}
+
+func (c *Cluster) createClientServiceLB() error {
```

Comment:
I think it's two separate issues here.

First of all, for internal code, I'm fine with the method name with "LB".

When using an etcd cluster, a name like "client service" is more intuitive to users. The client service for each member is ephemeral and for internal use only. We don't want user to use it.